scormxblock/scormxblock.py

```diff
@@ -470,6 +472,14 @@ def set_raw_scorm_status(self, request, suffix=''):
         # TODO: handle errors
         return Response(json.dumps(self.raw_scorm_status), content_type='application/json', charset='UTF-8')
 
+    @XBlock.handler
+    def get_scorm_completion(self, request, suffix=''):
+        completion = {'completion': self.scorm_progress or 0}
```

Comment:
No need for this or, since by default scorm_progress is 0.

Reply:
Just an extra layer of type checking